src/Widget/ProfileRowFormWidget.cpp

```diff
@@ -141,3 +173,39 @@ RowFormWidget::SaveValueFileReader(unsigned i, const char *registry_key)
   Profile::Set(registry_key, new_value2);
   return true;
 }
+
+
+bool 
+RowFormWidget::SaveValueMultiFileReader(unsigned i, const char *registry_key)
+{
+	const auto *dfe = (const MultiFileDataField *)GetControl(i).GetDataField();
+
+	std::vector<Path> new_values = dfe->GetPathFiles();
+
+	char new_output[MAX_PATH]; 
+
+	if (new_values.empty())
+		UnsafeCopyString(new_output, "");
+
+	for (auto value : new_values){
+
+		const auto contracted = ContractLocalPath(value);
+		if (contracted != nullptr)
+			value = contracted;
+
+		const WideToUTF8Converter value_to_add(value.c_str());
+		if (!value_to_add.IsValid())
+			continue;
+
+		strcat(new_output, value_to_add);
+		strcat(new_output, ",");
```

Comment:
Stack buffer overflow bug here. Never use `strcat()`!

Reply:
I'll change that to `strncat()`

Reply:
Do you know how `strncat()` works?
Do you know what it does when the buffer is too small?
(Spoiler: it omits the null terminator.)
Bottom line: don't use `strncat()` either. (Unless you really know what you're doing.)